python/lsst/meas/extensions/scarlet/deblend.py

```diff
@@ -39,14 +39,20 @@
 import lsst.afw.table as afwTable
 
 from .source import initSource, modelToHeavy
-from .blend import LsstBlend, checkBlendConvergence
-from .observation import LsstFrame, LsstObservation
 
 __all__ = ["deblend", "ScarletDeblendConfig", "ScarletDeblendTask"]
 
 logger = lsst.log.Log.getLogger("meas.deblender.deblend")
 
 
+def _checkBlendConvergence(blend, f_rel):
+    """Check whether or not a blend has converged
```

Comment:
I don't think this is true for private methods. It's certainly common practice to not include arguments and parameters for private methods and the only place this is mentioned in the dev guid is

```
def _private(self):
        """Do something internally.

        By default private members are not included in the HTML docs either.

        Notes
        -----
        Private members are any methods or attributes that start with an
        underscore and are *not* special. By default they are not included in
        the output.

        However, you should still provide docstrings for private members to
        document code for internal developers.
        """
        pass
```

Looking at the other method in that section (https://developer.lsst.io/python/numpydoc.html#complete-example-module) I feel like this implies that docstrings are needed but in a case where the parameters are obvious we don't need to be so verbose. But if you can find somewhere that says otherwise...